src/controllers/router-store/types.ts

```diff
@@ -61,12 +63,18 @@ type PrivateRouterActions = {
   requestRouteResources: () => RouterAction;
   listen: () => RouterAction;
   getContext: () => RouterAction;
-  pushQueryParam: (params: {
```

Comment:
We need to be cautious when removing part of the public API. Keen for @albertogasparin to provide input here.

I feel to be safe we should keep these actions, and just proxy them through to the `update` functions. Maybe log a warning in development too.

Reply:
these are part of the private router actions, are they still part of the public API?

Reply:
These are private so I think it's fine

Reply:
ah nice, i assumed they were public.
No worries!